Remotion/Data/DomainObjects.Validation.UnitTests/DomainObjectAttributesBasedValidationPropertyRuleReflectorTests/DomainObjectMixin_DOABCPRRTest.cs

```diff
@@ -371,21 +370,9 @@ public void GetRemovablePropertyValidators_MandatoryAttributeAndValidationMessag
     [Test]
     public void GetRemovablePropertyValidators_NullableStringPropertyAttribute ()
     {
-      var validationMessageStub = new Mock<ValidationMessage>();
-      _validationMessageFactoryStub
-          .Setup(
-              _ => _.CreateValidationMessageForPropertyValidator(
-                  It.IsAny<MaximumLengthValidator>(),
-                  PropertyInfoAdapter.Create(_mixinPropertyWithNullableStringPropertyAttribute)))
-          .Returns(validationMessageStub.Object);
-
       var result = _propertyWithNullableStringPropertyAttributeReflector.GetRemovablePropertyValidators().ToArray();
 
-      validationMessageStub.Setup(_ => _.ToString()).Returns("Stub Message");
-      Assert.That(result.Count(), Is.EqualTo(1));
-      Assert.That(result[0], Is.TypeOf(typeof(MaximumLengthValidator)));
-      Assert.That(((MaximumLengthValidator)result[0]).Max, Is.EqualTo(10));
-      Assert.That(((MaximumLengthValidator)result[0]).ValidationMessage.ToString, Is.EqualTo("Stub Message"));
+      Assert.That(result.Any, Is.False);
```

Comment:
```suggestion
      Assert.That(result, Is.Empty);
```

Reply:
Just mentioning this: `result.Any` passes a delagate to the Assertion method. Am surprised that the test works and isn't complaining about the mismatch in semantics. Normally, this syntax is only used for exception-semantics and the result isn't relevant. so, what does `Is.False` assert in this instance. So, even if we wouldn't change it to `Is.Empty`, we would still need to update to `result.Any()`